staging/src/k8s.io/client-go/tools/pager/pager.go

```diff
@@ -104,6 +146,8 @@ func (p *ListPager) List(ctx context.Context, options metav1.ListOptions) (runti
 			options.ResourceVersion = requestedResourceVersion
 			result, err := p.PageFn(ctx, options)
 			return result, paginatedResult, err
+		} else {
+			retries = retriesPerCall
```

Comment:
this doesn't need to be in an else clause since the if clause returns.

Reply:
done.